### PR TITLE
Ensure some commonly-used tools are present on bastion

### PR DESCRIPTION
**Pull request by meatballhat**

### CODE REVIEW

File: packer-scripts/minimize [hunk 1]
```diff
@@ -3,8 +3,7 @@
 set -o errexit
 set -o xtrace
 
-if [[ $PACKER_BUILDER_TYPE = googlecompute \
-  || $PACKER_BUILDER_TYPE = amazon-ebs ]]; then
+if [[ $PACKER_BUILDER_TYPE = googlecompute || $PACKER_BUILDER_TYPE = amazon-ebs ]]; then
```
Comment (meatballhat) [inline packer-scripts/minimize]: this was an automatic change from shfmt 😸

**Comment by igorwwwwwwwwwwwwwwwwwwww** [comment]
From the build logs:

```
$ ./bin/packer-build-install
curl -sSLo tmp/packer.zip 'https://releases.hashicorp.com/packer/0.10.1/packer_0.10.1_linux_amd64.zip'
mkdir -p ~/bin
unzip -d ~/bin tmp/packer.zip
Archive:  tmp/packer.zip
replace /home/travis/bin/packer? [y]es, [n]o, [A]ll, [N]one, [r]ename: 
The job exceeded the maximum time limit for jobs, and has been terminated.
```

**Comment by meatballhat** [comment]
@Lyoness r?

File: cookbooks/travis_internal_base/recipes/default.rb [hunk 2]
```diff
@@ -29,7 +29,7 @@
 include_recipe 'travis_sudo'
 
 package 'whois' do
-  action [:install, :upgrade]
+  action %i(install upgrade)
```
Comment (Lyoness) [inline cookbooks/travis_internal_base/recipes/default.rb]: i looked up this change for chef (ruby) syntax and still don't quite understand change.  All other changes LGTM :+1:

Comment (meatballhat) [inline cookbooks/travis_internal_base/recipes/default.rb]: It's a silly change on my part, imho. The resulting array of symbols is the same.

**Pull request merged**